src/main/java/org/commcare/formplayer/services/NewFormResponseFactory.java

```diff
@@ -156,7 +157,8 @@ public FormSession getFormSession(SerializableFormSession serializableFormSessio
                 storageFactory,
                 commCareSession,
                 formplayerRemoteInstanceFetcher,
-                formDefinitionService
+                formDefinitionService,
+                null
```

Comment:
we need to pass `window_width` from beans to all constructors otherwise specific request will not work. For ex. this will cause all `/incomplete-form` requests to have the param as `null`